include/veriblock/pop/crypto/progpow.hpp

```diff
@@ -6,9 +6,8 @@
 #ifndef VERIBLOCK_POP_CPP_PROGPOW_HPP
 #define VERIBLOCK_POP_CPP_PROGPOW_HPP
 
+#include <veriblock/pop/crypto/progpow/cache.hpp>
```

Comment:
for local includes use #include "file.h"

Reply:
fixed